test/e2e/images_test.go

```diff
@@ -422,7 +422,7 @@ LABEL "com.example.vendor"="Example Vendor"
 		Expect(result.OutputToStringArray()).To(Not(Equal(result1.OutputToStringArray())))
 	})
 
-	It("podman image prune --filter", func() {
+	FIt("podman image prune --filter", func() {
```

Comment:
@rhatdan Could you please drop changes in `test/e2e/images_test.go` this is not needed and I think this is causing CI to fail.